### DISCUSSION TITLE
Grid plugin for tinymce editor

### DISCUSSION THREAD

**Discussion by aukc1970** [givanz/Vvveb#235] [General]
Good afternoon! There is a wonderful grid plugin for tinymce editor is it possible to add it to the tinymce editor. Here are the links, please see which one is better for this cms.  
https://github.com/root913/tinymce-plugin-grid
https://github.com/tohen-media/tinymce-5-plugin-grid

**Comment by aukc1970** [comment]
Hello Givanz ! Please tell me how to add a plugin to tinymce editor in Vvveb CMS, where in which files should I write it? Thanks in advance for your answer!

**Comment by givanz** [comment]
Hi

You can load tinymce plugins by copying the tinymce plugin to `/public/js/libs/tinymce-dist/plugins/` then add it to plugins list and toolbar list in https://github.com/givanz/Vvveb/blob/master/public/js/libs/tinymce-options.js#L813 https://github.com/givanz/Vvveb/blob/master/public/js/libs/tinymce-options.js#L817

I added a simple plugin to load tinymce plugins with the grid plugin included, you can easily extend it to load other plugins
https://plugins.vvveb.com/product/tinymce-plugins

**Reply by aukc1970** [reply]
Thank you very much!
